tests/components/heos/test_init.py

```diff
@@ -80,7 +80,9 @@ async def test_async_setup_entry_loads_platforms(
 ) -> None:
     """Test load connects to heos, retrieves players, and loads platforms."""
     config_entry.add_to_hass(hass)
-    with patch.object(hass.config_entries, "async_forward_entry_setup") as forward_mock:
+    with patch.object(
+        hass.config_entries, "async_forward_entry_setups"
```

Comment:
We shouldn't really be patching the config entries helper in integrations.

Reply:
I 100% agree.  Unfortunately this pattern exists in too many places to keep it in scope for this PR.